### ISSUE TITLE
TInker's Compatibility with BYG and BiomesOP

### ISSUE THREAD

**Issue by KamieChimera** [SlimeKnights/TinkersConstruct#4862]
### Minecraft Version

1.18.2

### Forge Version

40.1.19

### Mantle Version

Mantle-1.18.2-1.9.20

### Tinkers' Construct Version

TConstruct-1.18.2-3.5.0.17

### Describe your issue

Using Tinker's Construct works in Vanilla Minecraft biomes, however when a smeltery is placed and set up in non-vanilla biomes, such as BYG (Biomes You'll Go) or Biomes OP. it fails to properly work, and will consume fuel at an abnormal rate, regardless of size.
When using the smeltery, the GUI lists 0 fuel, and likewise, no temperature. As a result, it has incorrect tick times for each lava bucket, consuming half a seared fuel tank with 3 ores smelted.

**To reproduce, Do the following -**
1. Go to a Vanilla biome, Set up a smeltery and right click the smeltery controller.
2. Once checked that it's working, head to a non-vanilla biome. A possible list is below --
     **- Allium Fields** 
     (https://oh-the-biomes-youll-go.fandom.com/wiki/Allium_Fields)
     **- Cherry Blossom Forest** 
     (https://oh-the-biomes-youll-go.fandom.com/wiki/Cherry_Blossom_Forest)
     **- Mangrove Marshes**
     (https://oh-the-biomes-youll-go.fandom.com/wiki/Mangrove_Marshes)
     **- Lavender Fields**
     (https://biomesoplenty.fandom.com/wiki/Lavender_Fields)
     **- Lush Savannah**
     (https://biomesoplenty.fandom.com/wiki/Lush_Savanna)
     **- Redwood Forest**
     (https://biomesoplenty.fandom.com/wiki/Redwood_Forest)
3. Right click the Smeltery Controller, and you should see 0 fuel being located inside the GUI. You should notice that the error is present if you see "No fuel" in the GUI 

**Expected behavior:**
See above. 
Expected behavior to be normal and properly consuming fuel.

**Screenshots**
**Biomes You'll Go F3 (Lush Stacks)**
![byglushstacksf3](https://user-images.githubusercontent.com/105662428/168641356-e5614298-c7dd-439e-970e-57fa35d828a9.png)
**BYG Smeltery Controller**
![bygsmeltery](https://user-images.githubusercontent.com/105662428/168641457-28a1728f-c4dc-4ca3-b307-eeb1a5a65726.png)
**Vanilla F3 (Meadow)**
![unknown](https://user-images.githubusercontent.com/105662428/168642327-8b1458bb-1d49-4ea8-93c7-89b401677c94.png)
**Vanilla Smeltery Controller**
![unknown](https://user-images.githubusercontent.com/105662428/168642430-df06cc39-a8c9-4156-abb0-c19f33360bcb.png)
**Biomes OP F3 (Bayou)**
![unknown](https://user-images.githubusercontent.com/105662428/168642929-c9be5abe-75e1-4b47-95a4-f0b2a323aeb0.png)
**Biomes OP Smeltery Controller**
![unknown](https://user-images.githubusercontent.com/105662428/168643340-844e3322-d5e9-4937-9e5b-df7112a7f93f.png)

### Other mods

Biomes O' Plenty
Biomes You'll Go
Tinker's Construct + Mantle

**Originally found in -** 
https://www.curseforge.com/minecraft/modpacks/tnp-limitless-5

### Tried reproducing with just Tinkers?

No

### Performance Enchancers

Optifine

### Searched for known issues?

Checked pinned issues, Searched open issues, Searched closed issues, Checked the FAQ

**Comment by KnightMiner** [comment]
The reason we ask you about performance enhancers is so you test without them. If this happens without optifine, you don't have to list it on the template. Go do as you are supposed to and test without optifine.

**Comment by KamieChimera** [comment]
I did do as I was supposed to. I tested two versions. The one using a
modpack with optifine included, and non optifine, which was stated with
mods.

When I tested with just BiomesOP and BYG it still occurred.

**Comment by KnightMiner** [comment]
To be clear, you are saying you can reproduce this issue with no mods besides Tinkers Construct and a biome mod installed? If so, please provide screenshots with that setup, and a full game log.

There is no sense showing the bug happens with a larger mod list if you know it happens with a smaller mod list. This is what I was saying about optifine, if the bug happens without optifine there is no need to mention it. If the bug happens without Natura's Aura there is no reason it should be included in the screenshots above.

**Comment by ShinyPorygon** [comment]
Unable to reproduce with Biomes O' Plenty or Biomes You'll Go. You probably have some other mod altering something. No extra fuel is used, everything renders fine, we need more information on what mods you are actually using. Otherwise there is nothing we can do to even remotely help.

**Comment by luckyu19** [comment]
Unable to reproduce the issue with the pack and steps provided.
I am using a 2x2x1 smeltery and tested in three separate biomes.
Tested without Optifine, of course.

**Comment by NytSong** [comment]
I'm actually experiencing something similar, just with a different version.  The fun bit is that it was working just fine for sometime, but now in our overworld base, it shows "no fuel found."  I rebuilt the whole thing and it still says "no fuel found."  But interestingly enough, one that we setup under a blaze spawner in the nether is still showing everything fine.

I hadn't thought to try in other biomes..  I'm using the FTB University 1.16 modpack that has Oh The Biomes You'll Go in it.

I couldn't pinpoint exactly when it stopped working either.  I thought I knew and tried restoring from a backup I took just before, but that didn't bring it back.  I hadn't done anything else noteworthy enough that I did an unscheduled manual backup between when it was working and then.

It's the fact that it was working just fine and suddenly it wasn't that makes me scratch my head and I'm even more confused about why it works fine in the nether still.
